docs/sphinx/_toc.yml.in

```
@@ -2,11 +2,20 @@
 # These comments will also be removed.
 root: index
 subtrees:
-  - numbered: False
+  - caption: Install
     entries:
-    - file: structure
-    - file: api
-    - file: environment_variables
+    - file: install/installation
+
+  - caption: API reference
+    entries:
+    - file: api-reference/c-interface-adaptors
+    - file: api-reference/environment_variables
+    - file: api-reference/api
+    
+  - caption: Contribution
```

Can we change "Contribution" to "How to" instead?  Same with card in index.rst?
```suggestion
  - caption: How to
```

---

@yhuiYH I am not sure if contribution should be a part of how to. How to should envelope stepwise information to achieve a specific goal. Also, it is better to keep information for users and contributors separately as those are different audiences. @lpaoletti Let me know the final decision and I shall implement it.